From 3e4f26a7c49fe12fe3125cbf47db3fbaf6ec90c3 Mon Sep 17 00:00:00 2001
From: Adrian DC <radian.dc@gmail.com>
Date: Sat, 9 Dec 2017 11:38:00 -0600
Subject: [PATCH 1/3] Dialer: disable anti-falsing for call answer screen

 * The anti-falsing implementation from HumanInteractionClassifier
    regularly prevents easy swipe to answer, requiring multiple
    attempts until accepted.

Change-Id: Iebad27f9da7bb8fea6fc663dc99ac6c17d94ed5f
(cherry picked from commit 583e42df0f6f02f1a6c05dbdc5559f42e99c8727)
---
 .../answermethod/FlingUpDownTouchHandler.java     | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/java/com/android/incallui/answer/impl/answermethod/FlingUpDownTouchHandler.java b/java/com/android/incallui/answer/impl/answermethod/FlingUpDownTouchHandler.java
index a21073d65..7abea5481 100644
--- a/java/com/android/incallui/answer/impl/answermethod/FlingUpDownTouchHandler.java
+++ b/java/com/android/incallui/answer/impl/answermethod/FlingUpDownTouchHandler.java
@@ -367,21 +367,6 @@ class FlingUpDownTouchHandler implements OnTouchListener {
   }
 
   private boolean isFalseTouch() {
-    if (falsingManager != null && falsingManager.isEnabled()) {
-      if (falsingManager.isFalseTouch()) {
-        if (touchUsesFalsing) {
-          LogUtil.i("FlingUpDownTouchHandler.isFalseTouch", "rejecting false touch");
-          return true;
-        } else {
-          LogUtil.i(
-              "FlingUpDownTouchHandler.isFalseTouch",
-              "Suspected false touch, but not using false touch rejection for this gesture");
-          return false;
-        }
-      } else {
-        return false;
-      }
-    }
     return !touchAboveFalsingThreshold;
   }
 
-- 
2.17.1

